lib/yarp.rb

```diff
@@ -311,6 +311,115 @@ def self.load(source, serialized)
   # This module is used for testing and debugging and is not meant to be used by
   # consumers of this library.
   module Debug
+    # For the given source, compiles with CRuby and returns a list of all of the
+    # sets of local variables that were encountered.
+    def self.cruby_locals(source)
+      verbose = $VERBOSE
+      $VERBOSE = nil
+
+      begin
+        locals = []
+        stack = [RubyVM::InstructionSequence.compile(source).to_a]
+
+        while (iseq = stack.pop)
+          if iseq[0] != :once
+            names = iseq[10]
+
+            # CRuby will push on a special local variable when there are keyword
+            # arguments. We get rid of that here.
+            names = names.grep_v(Integer)
+
+            # We don't support numbered local variables yet, so we get rid of
```

Comment:
If we're using the `TODO` convention later in this file, can we also use it here?